source/lib/position/constants/index.js

```diff
@@ -0,0 +1,5 @@
+import { ARROW } from '../constants/arrow';
+import { POSITION } from '../constants/position';
+import { ORIENTATION } from '../constants/orientation';
+
+export { ARROW, POSITION, ORIENTATION };
```

Comment:
Once all of the `source/lib/position/constants/*` files are merged back in, please rename to `source/lib/position/constants.js` to eliminate unnecessary file structure.